### PR TITLE
Start Docker if exists but not running

### LINKED ISSUE #204: Start Docker container if exists, but not running
### Description
If there is an existing Docker container, but it isn't running when we call `scooze setup docker` we should probably start it for the user.

### PR DESCRIPTION
**Pull request by fjork3**

### CODE REVIEW

File: src/scooze/console/cli.py [hunk 1]
```diff
@@ -31,7 +31,14 @@
...
         cli_words = name.split(" ")
-        module = import_module("scooze.console.commands." + ".".join(cli_words))
+        # We want `docker` in the setup command name, but need a different filename for import to work
+        # TODO: find a way to do this without ugly special casing
+        if cli_words[-1] == "docker":
+            cli_words[-1] = "setup-docker"
+            module = import_module("scooze.console.commands." + ".".join(cli_words))
+            cli_words[-1] = "docker"
+        else:
+            module = import_module("scooze.console.commands." + ".".join(cli_words))
```
Comment (iambroadband) [inline src/scooze/console/cli.py:41]: Is there an extra dot here?

Comment (iambroadband) [reply src/scooze/console/cli.py:41]: I guess not if we do the same elsewhere

Comment (fjork3) [reply src/scooze/console/cli.py:41]: Nope; `join` doesn't have one leading or trailing.

File: src/scooze/console/cli.py [hunk 2]
```diff
@@ -31,7 +31,14 @@
 def load_scooze_command(name: str) -> Callable[[], Command]:
     def _scooze_command_factory() -> Command:
         cli_words = name.split(" ")
-        module = import_module("scooze.console.commands." + ".".join(cli_words))
+        # We want `docker` in the setup command name, but need a different filename for import to work
+        # TODO: find a way to do this without ugly special casing
```
Comment (iambroadband) [inline src/scooze/console/cli.py]: Give this a ticket and put it here in parens.

**Review by iambroadband — DISMISSED** [review]

**Review by iambroadband — APPROVED** [review]

**Pull request merged**